### DISCUSSION TITLE
Estruturação de mentoria para organizações das PyBR

### DISCUSSION THREAD

**Discussion by cecivieira** [apyb/comunidade#10] [Geral]
Oi povo,

Com o objetivo de desenvolver uma mentoria estruturada para dar suporte às comunidades locais na organização da Python Brasil, nós da organização da PyBr2021 [iniciamos uma discussão](https://github.com/pythonbrasil/pybr2021-org/issues/347) sobre como a equipe do evento anterior conjuntamente com a APyB poderia apoiar a próxima equipe organizadora da Python Brasil. Compartilhamos a discussão aqui para que toda a comunidade possa se envolver.

**A ideia desse fio é estabelecer a estrutura de uma mentoria que deverá iniciar logo após a escolha da sede**: quantidade de encontros e conteúdo previsto para cada um. Essa estrutura será viabilizada através de encontros por video chamada e material escrito (manual e slides) e deverá ser replicada ano após ano.

As sugestões de modificação da proposta abaixo podem ser deixadas aqui neste fio, bem como nas reuniões que ocorrerão entre comunidade e APyB às quartas-feiras entre 8h30 e 9h30 (am), sendo a primeira em 23 fev. 2022.

----
# Mentoria para equipe organizadora da Python Brasil

- **Público alvo:** Big Kahuna e Equipe organizadora da próxima PyBr
- **Período para início:** Entre 24 e 22 meses antes da realização do evento
- **Responsáveis:** APyB e membras da organização do evento anterior

### 1. encontro: visão geral
**A ser realizado:** +- 22 meses antes do evento
1. Boas vindas
2. Apresentar agenda
3. O que é Python Brasil desde uma perspectiva de gestão (projeto de código aberto com uma membra fixa: a Gestora de Projeto)
4. Apresentar responsabilidade da APyB, como a associação pode dar suporte e possibilidade de pactuação do modelo de organização ([Como organizar eventos usando a infraestrutura da APyB](https://docs.google.com/document/d/1j0UwPbZrN1E8W3zdclzyIxaFPgztjLAATSdsPAizFX0/))
5. Papel da Big Kahuna (liderança de equipe, gestora de projeto, solicitar ajuda da Apyb, monitorar tamanho da equipe, realização de tarefas, etc.)
6. Linha do tempo:
    **1.** Pré-evento (ordem cronológica de atividades):
    
    **1.1.** 2 anos antes
        - Idealização do evento;
        - Convidar novas voluntárias (formação de equipe);
        - Estabelecer fluxo de comunicação e trabalho;
        - Escolha do local
        - Desenvolvimento de identidade visual;
        - Levantamento de custo para viabilização;
        - Criação de fluxo de caixa;
        - Planejar curadoria de conteúdo para o evento.  
              
    **1.2.** 1 ano antes 
        - Convidar novas voluntárias (durante a Python Brasil do ano anterior);
        - Criação do Plano de Patrocínio;
        - Formação de grupos de trabalho;
        - Contato com empresas (ativo e passivo);
        - Planejar campanhas publicitárias (ex.: lançamento do evento, chamada para patrocinadoras, submissão de palestras, etc.);
        - Estabelecer grade horária do evento; (com isso será possível determinar quantidade de palestras, jobfairs, sorteiros, etc); 
        - Pactuação contratual de fornecedores a partir da priorização de gastos e execução orçamentária.
        
    **2.** Execução do evento
    **3.** Pós-evento:
        - Execução orçamentária restante;
        - Fechamento do fluxo de caixa do evento e publicação no site;
        - Emissão de certificados;
        - Abertura de dados anonimizados no repositório [dados](https://github.com/pythonbrasil/dados)
        - Exclusão permanente de dados pessoais de participantes;
        - Envio de relatório de impacto para patrocinadoras;
        - Atualização do [Manual da Big Kahuna](manual.pythonbrasil.org.br.);
        - Redação de Post mortem
7. Ferramentas que podem ser reutilizadas, que já foram desenvolvidas e/ou testada pelas organizações anteriores:
    - Gestão de projeto
        - GitHub: gerenciamento de atividades e discussões;
    - Comunicação
        - Discord: comunicação online.
8. Conselhos finais
    - Comunicação:
        - Transparência como padrão;
        - Estabeleceder uma ótima comunicação assíncrona;
        - Estimular o uso das ferramentas de comunicação e gerenciamento estabelecida pela equipe;
        - Estimular uma comunicação direta e sincera;
        - Haver coversa de entendimento que todas as pessoas estão andando no mesmo sentido, no entanto discordancias acontecerão, cobranças também e isso não quer dizer que é um problema pessoal. As insatisfações precisam ser expostas para que haja a possibilidade de entendimento e resolução;
        - Registrar todo o processo de execução da atividade na plataforma escolhida e estimular colaboração de todas nas que demandarem discussão. Um projeto de código aberto sem documentação não serve de muita coisa e dificulta muito a contribuição de outras pessoas.

### 2. encontro: detalhamento sobre atividades a serem desenvolvidas com 2 anos de antecedencia
**A ser realizado:** 15 dias após o 1. encontro

- Idealização do evento:
    - Chuva de ideias para montar um evento ideal;
    - Designar atividades para a equipe entregar no próximo encontro: 
        - Esboço do evento (data, locais desejados, quantidade de participantes, quantidade de trilhas, acessibilidade do conteúdo para pessoas com deficiência auditiva, etc.). **Dica para organização:** fortemente levar em consideração a capacidade de trabalho da comunidade anfitriã do evento, seu engajamento na organização do evento, tempo disponível e possíveis ausências
    
- Formação de equipe; (importância da formação de uma equipe local e o papel da big kahuna nisso)
- Estabelecer fluxo de comunicação e trabalho (contar sobre experiência anterior, estabelecer ferramenta e fluxo);
- Escolha do local; (informar prazo para contratação do local e como APyB pode dar suporte a isso)
- Desenvolvimento de identidade visual;
- Levantamento de custo para viabilização; (tema a ser detalhado no próximo encontro)
- Criação de fluxo de caixa; (tema a ser detalhado nos próximos encontros)
- Planejar curadoria de conteúdo para o evento; (tema a ser detalhado nos proximos encontros)
 
### 3. encontro: mentoria para "Levantamento de custo para viabilização"
**A ser realizado:** 30 dias após o 2. encontro
- Revisar "Esboço do evento" proposto no encontro anterior
- Fornecer modelo de planilha para reunir todos os custos de infraestrutura e terceirização de serviços baseado em preços reais + 15%
- Designar atividades para a equipe entregar no próximo encontro: 
    - Considerando o "Esboço do projeto", iniciar o preenchimento da tabela com o detalhamento do que será necessário para a viabilização de cada atividade, ou seja, partir do macro e ir pensando cada pedaço do evento e no que precisará contratar para isso, por exemplo:

    - "Quero que no encerramento do evento tenha um espetáculo artístico"
        - De que tipo?
        - Precisará contratar artistas?
        - Precisará alugar equipamento para a execução do espetáculo?
        - Haverá alimentação para as artistas?
        - Não sabe? Blz, então se escolhe o grupo artístico entra em contato e faz todas essas perguntas.

### 4. encontro: mentoria sobre curadoria de conteúdo 
**A ser realizado:** 60 dias após o 3. encontro

- Revisar "Levantamento de custo para viabilização" proposto no encontro anterior;
- Definição do local do evento e contratação;
- Criação de fluxo de caixa;
- Planejar curadoria de conteúdo para o evento; (acredito que aqui pode haver uma atividade para entrega posterior)
    - Ferramenta a ser utilizada
    - Critérios ([ler](https://rgth.co/pt-br/blog/modelo-de-avaliacao-das-atividades-da-python-brasil/))
- Determinar prazo para entrega da identidade visual;
- Designar atividades para a equipe entregar no próximo encontro:
    - Site

### 5. encontro: acompanhamento das atividades
**A ser realizado:** 30 dias após o 4. encontro

- Entrega da identidade visual;
- Entrega de site;
- Acompanhamento da contratação do local para o evento.

### 6. encontro: detalhamento sobre atividades a serem desenvolvidas com 1 ano de antecedencia
**A ser realizado:** 30 dias após o 5. encontro

- Convidar novas voluntárias (durante a Python Brasil do ano anterior);
- Criação do Plano de Patrocínio; (tema a ser detalhado nos proximos encontros);
- Formação de grupos de trabalho;
- Contato com empresas (ativo e passivo);
- Planejar campanhas publicitárias; (tema a ser detalhado nos proximos encontros)
- Colocar site do evento no ar;
- Estabelecer grade horária do evento; (com isso será possível determinar quantidade de palestras, jobfairs, sorteiros, etc) (tema a ser detalhado nos proximos encontros)
- Pactuação contratual de fornecedores a partir da priorização de gastos e execução orçamentária (jamais pagar 100% do contrato adiantado. Explicitar apoio jurídico que APyB pode fornecer).
        
### 7. encontro: mentoria para "Criação do Plano de Patrocínio" (12 meses antes do evento)
**A ser realizado:** 30 dias após o 6. encontro

- Resgatar "Levantamento de custo para viabilização" proposto no encontro anterior;
- Disponibilizar modelo ou planos de eventos anteriores;
- Definir recebimento ou não de patrocínio estrangeiro;
- Dicas de como estabelecer valor das cotas e benefícios de cada uma;
- Calcular cotas estrangeiras, caso aplicável;
- Designar atividades para a equipe entregar no próximo encontro:
    - Desenvolvimento do plano de patrocínio

### 8. encontro: mentoria para "Campanhas publicitárias" e "Grade horária do evento"
**A ser realizado:** 30 dias após o 7. encontro

- Fornecer modelo de planejamento para campanhas publicitárias;
- Fornecer grade horária dos eventos anteriores e dar dicas de experiências exitosas; (com isso será possível determinar quantidade de palestras, jobfairs, sorteiros, etc)
- Designar atividades para a equipe entregar no próximo encontro:
    - Planejamento de campanhas publicitárias
    - Grade horária do evento

### 9. encontro: considerações finais
**A ser realizado:** 30 dias após o 8. encontro

- Revisão do "Planejamento de campanhas publicitárias" e "Grade horária do evento"
- Incentivar reuniões semanais da organização, caso ainda não estejam ocorrendo
 
 ### 10. encontro: pós-evento
**A ser realizado:** 1 semana após a realização da PyBR

- Execução orçamentária restante;
- Fechamento do fluxo de caixa do evento e publicação no site;
- Emissão de certificados;
- Abertura de dados anonimizados no repositório [dados](https://github.com/pythonbrasil/dados)
- Exclusão permanente de dados pessoais de participantes;
- Envio de relatório de impacto para patrocinadoras;
- Atualização do [Manual da Big Kahuna](manual.pythonbrasil.org.br.);
- Redação de Post mortem

**Comment by cecivieira** [comment]
> 2. encontro: detalhamento sobre atividades a serem desenvolvidas com 2 anos de antecedencia
Adicionar:
- APyB dar acesso a: pasta referente ao evento no Google Drive, repositório do site, repositório da organização, freshdesk, Canva

> 6. encontro: detalhamento sobre atividades a serem desenvolvidas com 1 ano de antecedencia
Adicionar:
- APyB dar acesso a: redes sociais

> 10. encontro: pós-evento
> A ser realizado: 1 semana após a realização da PyBR
>    Execução orçamentária restante;
>    Fechamento do fluxo de caixa do evento e publicação no site;
>    Emissão de certificados;
>    Abertura de [dados](https://github.com/pythonbrasil/dados) anonimizados no repositório dados
>    Exclusão permanente de dados pessoais de participantes;
>    Envio de relatório de impacto para patrocinadoras;
>    Atualização do [Manual da Big Kahuna](https://github.com/apyb/comunidade/discussions/manual.pythonbrasil.org.br.);
 >   Redação de Post mortem
 
 10. encontro: pós-evento
**A ser realizado:** 1 semana após a realização da PyBR
- Designar atividades para a equipe entregar no próximo encontro: 
    - Execução orçamentária restante;
    - Fechamento do fluxo de caixa do evento e publicação no site;
    - Emissão de certificados;
    - Abertura de dados anonimizados no repositório [dados](https://github.com/pythonbrasil/dados)
    - Exclusão permanente de dados pessoais de participantes;
    - Envio de relatório de impacto para patrocinadoras;
    - Atualização do [Manual da Big Kahuna](manual.pythonbrasil.org.br.);
    - Redação de Post mortem
 
11. encontro: finalização da organização
**A ser realizado:** 60 dias após o 10. encontro

Entrega das atividades designadas no encontro anterior.

**Comment by rougeth** [comment]
Sobre o objetivo da mentoria, eu vejo dois grandes pontos que são super valiosos:

1. Guiar o time e a Big Kahuna, que muitas vezes nunca organizaram um evento ou não sabem o que 
2. Permite a direção da APyB estar mais próxima da organização do evento, ajudando de forma mais ativa e supervisionando problemas macros (Lembrete: APyB é responsável pela Python Brasil acontecer).

Dito isso, importante considerar que:
- A Big Kahuna tem autonomia para fazer o evento da forma que desejar, usando ou não a mentoria.
- A direção da APyB pode desconhecer os detalhes da organização da Python Brasil. Pode acontecer um cenário em que ninguém da gestão vigente tenha participado da organização do evento.
- A direção da APyB não faz parte do time organizador da Python Brasil. Alguns tópicos da mentoria podem ser lidos de tal forma que parece que as tarefas são feitas em conjunto com a APyB. Alguns exemplos:
	- **Encontro 3 - Revisar “Esboço do evento” proposto no encontro anterior**: Eu não acho que a APyB deva dizer se o esboço está bom ou ruim. Na minha opinião, a APyB deve mostrar porque fazer o esboço do evento vale a pena e como isso pode impactar  os trabalhos futuros.
	-  **Encontro 4 - Designar atividades para a equipe entregar no próximo encontro**: Eu não acho viável a APyB gerenciar as tarefas da organização, esse é o papel da Big Kahuna.

Acho que poderíamos considerar um modelo para cada tópico nas discussões:
- Tópico X
	- O que é?
	- Como foi abordado nos anos anteriores?
	- Quais ferramentas e documentações podem ser usadas pela organização?
	- Dicas, opiniões, etc

Dessa forma, a APyB tem embasamento para guiar os encontros. Inclusive, idealmente, cada tópico teria um link para o manual.pythonbrasil.org.br com todos os detalhes necessários para essa discussão.

Pensando de forma prática e para tirarmos a ideia do papel, podemos fazer o primeiro teste com Caxias usando os tópicos que você listou.

**Comment by cecivieira** [comment]
> A direção da APyB não faz parte do time organizador da Python Brasil. Alguns tópicos da mentoria podem ser lidos de tal forma que parece que as tarefas são feitas em conjunto com a APyB. Alguns exemplos:
>    Encontro 3 - Revisar “Esboço do evento” proposto no encontro anterior: Eu não acho que a APyB deva dizer se o esboço está bom ou ruim. Na minha opinião, a APyB deve mostrar porque fazer o esboço do evento vale a pena e como isso pode impactar os trabalhos futuros.
>    Encontro 4 - Designar atividades para a equipe entregar no próximo encontro: Eu não acho viável a APyB gerenciar as tarefas da organização, esse é o papel da Big Kahuna.

As atividades e revisões propostas não foram no sentido de avaliar a qualidade, no entanto de acompanhar e mentorar seu desenvolvimento. Entendo que ainda que a gestão da APyB não tenha conhecimento suficiente para fazê-lo, é a entidade que deveria facilitar o contato com lideranças de eventos passados com o objetivo de que elas participem de determinado encontro.

A redação dos capítulos no manual.pythonbrasil.org.br deverá ser realizada no momento de preparação do material de suporte (capítulo e slides) para os encontros. Além disso, estimando o tempo para concluir a revisão da estrutura proposta e preparação do material para o 1. encontro, sugiro agendar sua realização para a semana do dia 15 de março de 2022 com a organização da PyBr2023 (Caxias do Sul).

Considerando que o intervalo proposto para cada encontro se manterá, teremos o seguinte cronograma:

| encontro | mês |
| ----------- | ------ |
| 1 | 15 março 2022 | 
| 2 | abril 2022  | 
| 3 | maio 2022  | 
| 4 | junho 2022  | 
| 5 | agosto 2022  | 
| 6 | setembro 2022  | 
| 7 | outubro 2022  | 
| 8 | novembro 2022  | 
| 9 | dezembro 2022  | 
| 10 | 1 semana após PyBR Caxias | 
| 11 | 60 dias após PyBR Caxias |

**Reply by adrianomargarin** [reply]
2022 vai ocorrer em Manaus

2023 será em Caxias

**Comment by gabubellon** [comment]
Colocando alguns pitacos, e primeiro sinalizando como essa ideia é muito importante.

- Vejo que muito material a ser utilizado no onboarding possa ser de certa forma estático; mesmo com pecularidades locais vejo que coisas possam ser gravadas ou escritas de forma a ser melhor consultada, evitar repetição e até mesmo ser usado como guia;
- Um ponto sobre acompahar/ajudar a orgaização da atual Python Brasil, o quão interessante seria ter "former Big Kahunas" ajudando ? Pessoas que foram Big Kahuna ou de organização anteriores e algumas vezes e tenham disponibilidade para alguma consultoria/ajuda necessária.

**Reply by rougeth** [reply]
> Vejo que muito material a ser utilizado no onboarding possa ser de certa forma estático; mesmo com pecularidades locais vejo que coisas possam ser gravadas ou escritas de forma a ser melhor consultada, evitar repetição e até mesmo ser usado como guia;

Boa, eu entendi que o objetivo é popular o manual com conteúdo gerado a partir das primeiras sessões de mentoria.

> Um ponto sobre acompahar/ajudar a orgaização da atual Python Brasil, o quão interessante seria ter "former Big Kahunas" ajudando ? Pessoas que foram Big Kahuna ou de organização anteriores e algumas vezes e tenham disponibilidade para alguma consultoria/ajuda necessária.

Tenho mais lembranças de big kahunas que, após organizarem o evento, pararam de participar da comunidade por um tempo (alguns nem voltaram) do que aqueles que continuaram ajudando a edição seguinte. Eu acho que tornar isso algo oficial/necessário, aumenta muito o peso que já grande de organizar o evento.

**Reply by gabubellon** [reply]
`Tenho mais lembranças de big kahunas que, após organizarem o evento, pararam de participar da comunidade por um tempo (alguns nem voltaram) do que aqueles que continuaram ajudando a edição seguinte` Frase interessante que acho que vale uma conversa/tópico só sobre isso pra entender melhor (e não é o foco aqui)

**Comment by cecivieira** [comment]
Trecho revisado e concluído na reunião de 23 fev 2022:
(Ver documento completo em: https://hackmd.io/@IAA8H7ZURCejO-4Zz31Dug/rkuWWzik5

### 1. encontro: visão geral
**A ser realizado:** +- 22 meses antes do evento
1. Boas vindas
2. Apresentar agenda
3. O que é Python Brasil desde uma perspectiva de gestão (projeto de código aberto com uma membra fixa: a Gestora de Projeto)
4. Apresentar responsabilidade da APyB, como a associação pode dar suporte e possibilidade de pactuação do modelo de organização ([Como organizar eventos usando a infraestrutura da APyB](https://docs.google.com/document/d/1j0UwPbZrN1E8W3zdclzyIxaFPgztjLAATSdsPAizFX0/))
5. Papel da Big Kahuna (liderança de equipe, gestora de projeto, solicitar ajuda da Apyb, monitorar tamanho da equipe, realização de tarefas, etc.)
6. Linha do tempo:
    **1.** Pré-evento (ordem cronológica de atividades):
    
    **1.1.** 2 anos antes
        - Idealização do evento;
        - Convidar novas voluntárias (formação de equipe);
        - Estabelecer fluxo de comunicação e trabalho;
        - Escolha do local
        - Desenvolvimento de identidade visual;
        - Levantamento de custo para viabilização;
        - Criação de fluxo de caixa;
        - Planejar curadoria de conteúdo para o evento.  
              
    **1.2.** 1 ano antes 
        - Convidar novas voluntárias (durante a Python Brasil do ano anterior);
        - Criação do Plano de Patrocínio;
        - Formação de grupos de trabalho;
        - Contato com empresas (ativo e passivo);
        - Planejar campanhas publicitárias (ex.: lançamento do evento, chamada para patrocinadoras, submissão de palestras, etc.);
        - Estabelecer grade horária do evento; (com isso será possível determinar quantidade de palestras, jobfairs, sorteiros, etc); 
        - Pactuação contratual de fornecedores a partir da priorização de gastos e execução orçamentária.
        
    **2.** Execução do evento
    **3.** Pós-evento:
        - Execução orçamentária restante;
        - Fechamento do fluxo de caixa do evento e publicação no site;
        - Emissão de certificados;
        - Abertura de dados anonimizados no repositório [dados](https://github.com/pythonbrasil/dados)
        - Exclusão permanente de dados pessoais de participantes;
        - Envio de relatório de impacto para patrocinadoras;
        - Atualização do [Manual da Big Kahuna](manual.pythonbrasil.org.br.);
        - Redação de Post mortem.
7. Ferramentas necessárias e sugestões:
    - Gestão de projeto
        [descrever o uso]
        Sugestão: GitHub
    - Comunicação interna
        [descrever o uso]
        Sugestão: Discord
    - Comunicação externa
        Sugestão: Freshdesk e Hootsuite
    - Inscrição e submissão de atividades
    - Emissão e envio de certificados
        Sugestão: [Gerador de certificados](https://github.com/gabubellon/certificates-google-sheets-generator) e MailChimp
8. Conselhos finais
    - Comunicação:
        - Transparência como padrão;
        - Estabeleceder uma ótima comunicação assíncrona;
        - Estimular o uso das ferramentas de comunicação e gerenciamento estabelecida pela equipe;
        - Estimular uma comunicação direta e sincera;
        - Haver coversa de entendimento que todas as pessoas estão andando no mesmo sentido, no entanto discordancias acontecerão, cobranças também e isso não quer dizer que é um problema pessoal. As insatisfações precisam ser expostas para que haja a possibilidade de entendimento e resolução;
        - Registrar todo o processo de execução da atividade na plataforma escolhida e estimular colaboração de todas nas que demandarem discussão. Um projeto de código aberto sem documentação não serve de muita coisa e dificulta muito a contribuição de outras pessoas.

**Comment by cecivieira** [comment]
Trecho revisado e concluído na reunião de 09 mar. 2022:
(Ver documento completo em: https://hackmd.io/@IAA8H7ZURCejO-4Zz31Dug/rkuWWzik5)

### 2. encontro: detalhamento sobre atividades a serem desenvolvidas com 2 anos de antecedencia
**A ser realizado:** 15 dias após o 1. encontro
- Idealização do evento:
    - Chuva de ideias para montar um evento ideal (argumentar o porquê pensar nisso agora);
    - Designar atividades para a equipe entregar no próximo encontro: 
        - Esboço do evento (data, locais desejados, quantidade de participantes, quantidade de trilhas, acessibilidade do conteúdo para pessoas com deficiência auditiva, etc.). **Dica para organização:** fortemente levar em consideração a capacidade de trabalho da comunidade anfitriã do evento, seu engajamento na organização do evento, tempo disponível e possíveis ausências
    
- Formação de equipe; (importância da formação de uma equipe local e o papel da big kahuna nisso)
- Estabelecer fluxo de comunicação e trabalho (contar sobre experiência anterior, estabelecer ferramenta e fluxo para gestão das atividades, listar tópicos importantes na escolha da ferramenta);
- Escolha do local; (realizar escolha apartir do esboço do evento, falar sobre possibilidades (instituição pública e privada), sugerir criação de plano B, estabelecer prazo para contratação (formalização documental completa e concluída) e informar como APyB pode dar suporte a isso)
- Desenvolvimento de identidade visual;
- Levantamento de custo para viabilização; (tema a ser detalhado no próximo encontro)
- Criação de fluxo de caixa; (tema a ser detalhado nos próximos encontros)
- Planejar curadoria de conteúdo para o evento; (tema a ser detalhado nos proximos encontros, visão geral sobre cultura da comunidade (ver artigo Rougeth))
 
### 3. encontro: mentoria para "Levantamento de custo para viabilização"
**A ser realizado:** 30 dias após o 2. encontro
- Acompanhamento APyB
    - "Esboço do evento" proposto no encontro anterior
- Fornecer modelo de planilha para registrar fluxo de caixa;
- Fornecer modelo de planilha para reunir todos os custos de infraestrutura e terceirização de serviços baseado em preços reais + 15%
- Designar atividades para a equipe entregar no próximo encontro: 
    - Considerando o "Esboço do projeto", iniciar o preenchimento da tabela com o detalhamento do que será necessário para a viabilização de cada atividade, ou seja, partir do macro e ir pensando cada pedaço do evento e no que precisará contratar para isso, por exemplo:

    - "Quero que no encerramento do evento tenha um espetáculo artístico"
        - De que tipo?
        - Precisará contratar artistas?
        - Precisará alugar equipamento para a execução do espetáculo?
        - Haverá alimentação para as artistas?
        - Não sabe? Blz, então se escolhe o grupo artístico entra em contato e faz todas essas perguntas.
    - Argumentar o motivo de realizar esse levantamento com tanta antecedência, cotas de patrocínio.
